lab_1/main.py

```diff
@@ -30,7 +44,17 @@ def calculate_frequencies(tokens: list) -> dict or None:
     :param tokens: a list of tokens
     :return: a dictionary with frequencies
     """
-    pass
+    if isinstance(tokens, list):
```

Comment:
None check

Reply:
here and further

Reply:
Moved None check to the beginning everywhere